lib/python_prefida/colored.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+#from termcolor import colored as col
+
+#def colored(string, color):
+#    return col(string, color=color)
+
+import os
+
+# The following is taken directly from termcolor package
```

Comment:
Its OK to use this code since the termcolor package use the MIT license (same as FIDASIM) but to be in full compliance with its licence you need include a copy of the licence 

As stated in the MIT licence
>The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.